lib/modules/versioning/lambda-node/schedule.ts

```diff
@@ -0,0 +1,30 @@
+import dataFiles from '../../../data-files.generated';
+import { isStable } from '../node';
+
+interface LambdaSchedule {
+  cycle: string;
+  releaseLabel: string;
+
+  /**
+   * Either `true` if currently in support or a string indicating the date at which support will end
+   */
+  support: true | string;
+}
+
+export type LambdaData = Record<string, LambdaSchedule>;
+
+const lambdaSchedule: LambdaData = JSON.parse(
+  dataFiles.get('data/lambda-node-js-schedule.json')!,
+);
+
+export function findLambdaScheduleForVersion(
+  version: string,
+): LambdaSchedule | null {
+  const majorVersionMatch = /^v?([0-9]+)\./.exec(version);
+
+  if (!isStable(version)) {
+    return null;
+  }
+
+  return lambdaSchedule[majorVersionMatch![1]] ?? null;
```

Comment:
are you sure it always matched when coming here?

Reply:
If it's not matched, the `isStable(version)` check will fail first and return null. But good point, that's non-obvious and not within this function. I've  added a check to see if it was matched and verified that there's already a test `isStable("Iron")` that hits that return and is testing it